client/src/app/site/interaction/services/applause.service.ts

```diff
@@ -50,26 +54,38 @@ export class ApplauseService {
         return this.applauseTypeObservable.pipe(map(type => type === ApplauseType.bar));
     }
 
+    public get showApplauseLevelObservable(): Observable<boolean> {
+        return combineLatest([this.showApplauseLevelConfigObservable, this.applauseLevelObservable]).pipe(
+            map(([enabled, amount]) => {
+                return enabled && amount > 0;
+            })
+        );
```

Comment:
This thing is a little bit special. It does the following:
combine and evaluate the config "show applause amount" and the current "applause level". If the config is set and the applause was bigger than 0, show this (single) observable would emit `true`.

https://rxjs.dev/api/index/function/combineLatest